Parser/JsonMap.php

```diff
@@ -46,7 +46,18 @@ public function __construct(Config $config, LoggerInterface $logger, ParserInter
         $mappers = [];
         foreach ($config->getAttribute('mappings') as $type => $mapping) {
             if (empty($mapping)) {
-                throw new UserException("Empty mapping for '{$type}' in config.");
+                throw new UserException(sprintf(
+                    "Empty mapping for '%s' data type in 'mappings' config.",
+                    $type
+                ));
+            }
+
+            if (!is_array($mapping)) {
```

Comment:
Ak `$mapping` nie je pole, tak to nespadne, ale vygeneruje sa `UserError`.

Ostatne hlasky som upravil, aby boli viac jednoznacne.